src/seedu/addressbook/commands/Sort.java

```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.commands;
+
+import java.util.Collections;
+import java.util.List;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+public class Sort extends Command {
+	public static final String COMMAND_WORD = "sort entries";
+	public static final String MESSAGE_USAGE = COMMAND_WORD + ":\n" 
+            + "Displays all the entries in alphabetical order.\n\t";
+	public static final String MESSAGE_SUCCESS = "Sorted List: %1$s";
+	
+	
+	 
+	 @Override
+	  public CommandResult execute() {
+	        List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
+	        Collections.sort((List<ReadOnlyPerson>) allPersons);
```

Comment:
`ReadOnlyPerson` needs to be [Comparable](https://docs.oracle.com/javase/8/docs/api/java/lang/Comparable.html) in order to be able to be sorted with `Collections.sort()`.

Doing a search on Google for `Java Collections.sort comparable` gives me these links:

* https://www.mkyong.com/java/java-object-sorting-example-comparable-and-comparator/
* https://docs.oracle.com/javase/tutorial/collections/interfaces/order.html